tests/github/experimental/workflow/FederatedRuntime/testcase_private_attributes/workspace/testflow_privateattributes.ipynb

```diff
@@ -0,0 +1,411 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "dc13070c",
+   "metadata": {},
+   "source": [
+    "# Testcase: Private Attributes Jupitar Notebook"
```

Comment:
Nit - Jupyter 😉

Reply:
Removed the 'Jupyter notebook' for consistency with other test cases.